.github/workflows/github-ci.yml

```diff
@@ -1,27 +1,25 @@
 name: build/deploy opensim-python-runner
 
 on:
+  workflow_dispatch:
```

Comment:
so this entry is to trigger it manually?

Reply:
Yes. I found it useful while testing CI. I can remove it, if you see some potential drawbacks.

Reply:
no, no ... i was curious. Never saw it before